modules/commerce_bluesnap_currency/src/EventSubscriber/ExchangeRateBluesnap.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Drupal\commerce_bluesnap_currency\EventSubscriber;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\commerce_currency_resolver\ExchangeRateEventSubscriberBase;
+use Drupal\Component\Serialization\Json;
```

Comment:
**CHANGE REQUEST**

Please follow our conventions for `use` statements i.e. modules first then core:

```
use Drupal\commerce_currency_resolver\ExchangeRateEventSubscriberBase;
use Drupal\Core\Config\ConfigFactoryInterface;
use Drupal\Component\Serialization\Json;
```

Reply:
Modified

Reply:
Sorry, my bad. Within `use` statement groups (i.e. modules or core) the `use` statements need to be alphabetically sorted i.e.

```
use Drupal\commerce_currency_resolver\ExchangeRateEventSubscriberBase;
use Drupal\Component\Serialization\Json;
use Drupal\Core\Config\ConfigFactoryInterface;
```

Reply:
`Json` and `ConfigFactoryInterface` are core right? So in alphabetic sort the order would be
`
use Drupal\Core\Config\ConfigFactoryInterface;
use Drupal\Component\Serialization\Json;
` - Right?

Reply:
`Component` goes before `Core`

Reply:
Right. Made the change